### PR TITLE
WRQ-7108: Remove unnecessary vendor prefixes

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Sandstone has CSS vendor prefixes. Most of them are not required anymore.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Considering browser versions we support, I removed unnecessary vendor prefixes.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-7108

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim (jiye.kim@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
@vJIYEv , Please update your branch with the latest develop.

File: samples/sampler/src/ThemeEnvironment/ThemeEnvironment.module.less [hunk 1]
```diff
@@ -6,9 +6,6 @@
 // from ~@enact/ui/styles/core.less. Can't import global mixins.
 .enact-selectable() {
 	cursor: auto;
-	-ms-user-select: element;
-	-webkit-user-select: text;
```
Comment (MikyungKim) [inline samples/sampler/src/ThemeEnvironment/ThemeEnvironment.module.less:10]: As I mentioned in the enact PR, I think we should keep `-webkit-` prefix one.

Comment (vJIYEv) [reply samples/sampler/src/ThemeEnvironment/ThemeEnvironment.module.less:10]: OK.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

**Pull request merged**